snuba/datasets/partitioning.py

```diff
@@ -3,4 +3,32 @@
 should be stored. These do not require individual physical partitions but allow
 for repartitioning with less code changes per physical change.
 """
+from snuba.datasets.storages.storage_key import StorageKey
+
 SENTRY_LOGICAL_PARTITIONS = 256
+
+
+def map_org_id_to_logical_partition(org_id: int) -> int:
+    """
+    Maps an org_id to a logical partition. Since SENTRY_LOGICAL_PARTITIONS is
+    fixed, an org id will always be mapped to the same logical partition.
+    """
+    return org_id % SENTRY_LOGICAL_PARTITIONS
+
+
+def map_logical_partition_to_slice(storage: StorageKey, logical_partition: int) -> int:
+    """
+    Maps a logical partition to a slice.
+    """
+    from snuba.settings import LOGICAL_PARTITION_MAPPING
+
+    return LOGICAL_PARTITION_MAPPING[storage.value][logical_partition]
```

Comment:
This could potentially throw an exception in cases where someone calls this for a storage which is not partitioned. We should wrap this in a try/except and raise an exception

Reply:
If I understand correctly, this will only be called in contexts where the storage is sliced?

Reply:
I think I'm going to add assertions, this would be a misconfiguration and we shouldn't try to recover from that as an exception.